projects/plugins/social/src/js/components/types/types.ts

```diff
@@ -0,0 +1,39 @@
+type JetpackSettingsSelectors = {
+	getJetpackSettings: () => {
+		publicize_active: boolean;
+		show_pricing_page: boolean;
+		showNudge: boolean;
```

Comment:
I know it wasn't introduced in this PR (and should be fixed in a separate PR) but we should really be consistent here with casing 😬